crates/ruma-events/src/room.rs

```diff
@@ -20,6 +20,7 @@ pub mod encryption;
 pub mod guest_access;
 pub mod history_visibility;
 pub mod join_rules;
+pub mod matrix_rtc;
```

Comment:
The module structure in `ruma-events` mirrors the event-type strings. This new module should not exist, instead the definition of the `m.call` event should move to the existing `call` module (outside `room`), and the member event should move to a new module `call::member`.